crates/handlers/src/graphql/mod.rs

```diff
@@ -218,7 +218,7 @@ async fn get_requester(
         };
 
         // If there is a user for this session, check that it is not locked
-        let user_valid = user.as_ref().map_or(false, User::is_valid);
+        let user_valid = user.as_ref().map_or(true, User::is_valid);
```

Comment:
Because that was a mistake in the first place (but I did not bother fixing the original commit, sorry about that).

The logic is that if there is a user, it should be checked through the `User::is_valid`, and if there is no user, it's always "valid"